src/overlays/actors/ovl_Obj_Hgdoor/z_obj_hgdoor.c

```diff
@@ -122,50 +122,52 @@ void func_80BD42C0(ObjHgdoor* this, PlayState* play) {
     }
 }
 
-void func_80BD433C(ObjHgdoor* this) {
-    this->unk166 = 0x63;
-    this->actionFunc = func_80BD4358;
+void ObjHgdoor_SetupCsAction(ObjHgdoor* this) {
+    this->csAction = 0x63;
+    this->actionFunc = ObjHgdoor_HandleCsAction;
 }
 
-void func_80BD4358(ObjHgdoor* this, PlayState* play) {
+void ObjHgdoor_HandleCsAction(ObjHgdoor* this, PlayState* play) {
     s32 actionIndex;
 
     if (Cutscene_CheckActorAction(play, 483)) {
         actionIndex = Cutscene_GetActorActionIndex(play, 483);
-        if (this->unk166 != play->csCtx.actorActions[actionIndex]->action) {
-            this->unk166 = play->csCtx.actorActions[actionIndex]->action;
+        if (this->csAction != play->csCtx.actorActions[actionIndex]->action) {
+            this->csAction = play->csCtx.actorActions[actionIndex]->action;
             switch (play->csCtx.actorActions[actionIndex]->action) {
                 case 1:
                     Actor_PlaySfxAtPos(&this->dyna.actor, NA_SE_EV_WOOD_DOOR_OPEN_SPEEDY);
                     if ((this->dyna.actor.parent != NULL) && (this->dyna.actor.parent->id == ACTOR_EN_HG)) {
                         this->dyna.actor.parent->colChkInfo.health = 1;
                     }
-                    D_80BD4690 = 1;
+                    DoorOpenFlag = 1;
                     break;
                 case 2:
                     break;
             }
         }
-        if ((D_80BD4690 == 1) && (func_80BD44D0(this, play))) {
-            func_80BD4460(this);
+        if ((DoorOpenFlag == 1) && (ObjHgdoor_RotateDoor(this, play))) {
+            ObjHgdoor_SetupStopCs(this);
         }
     } else {
-        this->unk166 = 0x63;
+        this->csAction = 0x63;
     }
 }
 
-void func_80BD4460(ObjHgdoor* this) {
+void ObjHgdoor_SetupStopCs(ObjHgdoor* this) {
     this->timer = 0;
-    this->actionFunc = func_80BD4478;
+    this->actionFunc = ObjHgdoor_StopCs;
 }
 
-void func_80BD4478(ObjHgdoor* this, PlayState* play) {
+void ObjHgdoor_StopCs(ObjHgdoor* this, PlayState* play) {
     if (this->timer++ > 80 && !ActorCutscene_GetCanPlayNext(this->cutscene)) {
         ActorCutscene_Stop(this->cutscene);
     }
 }
 
-s32 func_80BD44D0(ObjHgdoor* this, PlayState* play) {
+// @brief Function to increment the rotation angle of the door
+// @return true when door is fully open, false otherwise
+s32 ObjHgdoor_RotateDoor(ObjHgdoor* this, PlayState* play) {
```

Comment:
This is the way it is done everywhere else in the codebase. Vscode suggests to use this format when doing `/**` and then pressing enter
```suggestion
/**
 * @brief Function to increment the rotation angle of the door
 * @return true when door is fully open, false otherwise
 */
s32 ObjHgdoor_RotateDoor(ObjHgdoor* this, PlayState* play) {
```